variables.tf

```diff
@@ -67,6 +67,13 @@ variable "has_external_secrets" {
   description = "Whether the Kubernetes Metrics Server will be installed."
 }
 
+
+variable "has_argocd" {
```

Comment:
We need to add another variable that would set the argocd ALB ingress group name:
[https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/guide/ingress/annotations/#group.name](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/guide/ingress/annotations/#group.name)

default = "internal"